frontend/src/Form/beta/ModelCardForm.tsx

```diff
@@ -0,0 +1,120 @@
+  splitSchema,
+  setSplitSchema,
+  canEdit = false,
+}: {
+  splitSchema: SplitSchemaNoRender
+  setSplitSchema: Dispatch<SetStateAction<SplitSchemaNoRender>>
+  canEdit?: boolean
+}) {
+  const [activeStep, setActiveStep] = useState(0)
+  // TODO - add validation
+  const [_openValidateError, _setOpenValidateError] = useState(false)
+
+  const theme = useTheme()
+
+  const currentStep = splitSchema.steps[activeStep]
+
+  if (!currentStep) {
+    return null
+  }
+
+  const onFormChange = (form) => {
+    if (form.schema.title !== currentStep.schema.title) {
+      return
+    }
+
+    setStepState(splitSchema, setSplitSchema, currentStep, { ...currentStep.state, ...form.formData })
+  }
+
+  function DescriptionFieldTemplate(_props: DescriptionFieldProps) {
+    return <></>
+  }
+
+  return (
+    <Stack
+      direction={{ xs: 'column', sm: 'row' }}
+      spacing={4}
+      justifyContent='left'
+      divider={<Divider flexItem orientation='vertical' />}
+      sx={{ width: '100%' }}
+    >
+      <Box>
+        <Stepper
+          id='form-page-stepper'
+          activeStep={activeStep}
+          nonLinear
+          alternativeLabel
+          orientation='vertical'
+          connector={<Nothing />}
+          sx={{ minWidth: 'max-content' }}
+        >
+          {splitSchema.steps.map((step, index) => (
+            <MaterialStep key={step.schema.title}>
+              <StepButton sx={{ p: 0, m: 0 }} onClick={() => setActiveStep(index)} icon={<Nothing />}>
+                <StepLabel
+                  sx={{
+                    padding: 0,
+                    '& .MuiStepLabel-label': {
+                      fontSize: '16px !important',
+                    },
+                    '& .Mui-active': {
+                      borderBottomStyle: 'solid',
+                      color: `${theme.palette.primary.main} !important`,
+                      borderColor: `${theme.palette.secondary.main} !important`,
+                    },
+                  }}
+                >
+                  {step.schema.title}
+                </StepLabel>
+              </StepButton>
+            </MaterialStep>
+          ))}
+        </Stepper>
+      </Box>
+
+      <Form
+        schema={currentStep.schema}
+        formData={currentStep.state}
+        onChange={onFormChange}
+        validator={validator}
+        noValidate
+        widgets={{
+          nothing: Nothing,
+          customTextInput: CustomTextInput,
+          tagSelector: TagSelector,
+        }}
+        uiSchema={currentStep.uiSchema}
+        liveValidate={currentStep.shouldValidate}
+        omitExtraData
+        disabled={!canEdit}
+        liveOmit
+        formContext={{ editMode: canEdit, formSchema: currentStep.schema }}
+        templates={
+          !canEdit
+            ? {
+                DescriptionFieldTemplate,
+              }
+            : {}
+        }
+      >
+        {/* eslint-disable-next-line react/jsx-no-useless-fragment */}
```

Comment:
Can this be changed to not pass in children? E.g. `<Form />`

Reply:
This is needed as without it, it will render the submit buttons which we no longer need. I'm not sure why this is the case, but this has been the same since we used the library in old version.